Set-1/Solved Programs/BrainCar/TestCaseForBrainCar.java

```diff
@@ -0,0 +1,29 @@
+package com.braincar;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+
+public class TestCaseForBrainCar {
+
+	@Test
+	public  void positive_test_For_BrainCarName() {

```

Comment:
name should follow camel case. also positive test for something doesn't give any info. test name should be proper and meaningful like testBrainCarNameWhen<situation> .

Reply:
ok i will do changes